src/packages/Highlighter/Highlighter.ts

```diff
@@ -196,14 +196,18 @@ export class Highlighter {
         });
     }
 
-    public find(terms: string[], highlighterCoreOptions?: HighlighterCoreOptions) {
+    public find(
+        terms: string[],
+        highlighterCoreOptions?: HighlighterCoreOptions,
+        highlightStyles?: CSSStyleDeclaration
+    ) {
         const matchedTerms: HighlighterResultItem[] = [];
         const notMatchedTerms: HighlighterResultItem[] = [];
         const sortedTerms = HighlighterCore.sortTerms(terms);
 
         sortedTerms.forEach((term, index) => {
             let amount = 0;
-            const styles = Highlighter.generateHighlightStyles(index);
+            const styles = highlightStyles || Highlighter.generateHighlightStyles(index);
```

Comment:
what do you think about overriding styles, not replacing:
```
const styles = {
  ...Highlighter.generateHighlightStyles(index),
  ...highlightStyles,
};
```

Reply:
`generateHighlightStyles` returns the bunch of styles that actually redundant for the current purpose - having the only background color. 
```
    border-radius: 1px;
    display: inline-block;
    text-indent: initial;
    background-color: rgb(214, 244, 129);
```

Reply:
` border-radius: 1px;` is'n it useful one?
So if we just need to change color - we also have to send other styles to have consistent style everywhere in TK

Reply:
The `border-radius` looks fancy for the overview profile where we have bright colors in the background and use the border for selected terms and elements should be `inline-block`. For yellow highlighting (string), it doesn't make sense.